src/Microsoft.TestPlatform.AdapterUtilities/ManagedNameUtilities/ManagedNameParser.cs

```diff
@@ -66,7 +64,7 @@ public static void ParseManagedTypeName(string managedTypeName, out string names
     /// <exception cref="InvalidManagedNameException">
     /// Thrown if <paramref name="managedMethodName"/> contains spaces, incomplete, or the arity isn't numeric.
     /// </exception>
-    public static void ParseManagedMethodName(string managedMethodName, out string methodName, out int arity, out string[] parameterTypes)
+    public static void ParseManagedMethodName(string managedMethodName, out string methodName, out int arity, out string[]? parameterTypes)
```

Comment:
`parameterTypes` can't ever be `null`, if it doesn't exactly have the elements we expect we throw.